kratos/python/add_model_part_to_python.cpp

```diff
@@ -726,8 +727,22 @@ void AddModelPartToPython(pybind11::module& m)
     PointerVectorSetPythonInterface<ModelPart::MasterSlaveConstraintContainerType>().CreateInterface(m,"MasterSlaveConstraintsArray");
 
     class_<ModelPart, Kratos::shared_ptr<ModelPart>, DataValueContainer, Flags >(m,"ModelPart")
-        .def(init<std::string const&>())
-        .def(init<>())
+        .def(init([](const std::string& name) { 
+                    std::cout << "************************************************************" << std::endl;
+                    std::cout << "************************************************************" << std::endl;
+                    std::cout << "************************************************************" << std::endl;
+                    std::cout << "USING OLD DEPRECATED CONSTRUCTOR OF MODELPART" << std::endl;
+                    std::cout << "THIS WILL BE REMOVED ON NOV 1 2018" << std::endl;
+                    std::cout << "the call to ModelPart(" << name << " ) " << std::endl;
+                    std::cout << "should be substituted by current_model.CreateModelPart(" << name << ") " << std::endl;
+                    std::cout << "************************************************************" << std::endl;
+                    std::cout << "************************************************************" << std::endl;
+                    std::cout << "************************************************************" << std::endl;
+                    static Model static_model; //NOT NICE! to be removed!!
+                    return std::make_shared<ModelPart>(name, static_model);
+                }
```

Comment:
I would suggest to use `KRATOS_WARNING` instead of `std::cout`.

Meanwhile, The message is too big and I don't think its size helps in its recognition....